database/migrations/2022_01_06_104351_messages_add_attachment_id_field.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use RonasIT\Support\Traits\MigrationTrait;
+
+class MessagesAddAttachmentIdField extends Migration
+{
+    use MigrationTrait;
+
+    public function up()
+    {
+        Schema::table('messages', function (Blueprint $table) {
+            $table->integer('attachment_id')->nullable();
+            $table->foreign('attachment_id')->references('id')->on('media')->onDelete('set null');
```

Comment:
let's move media table name to the config and move this fields to the first create messages table migration